src/index.ts

```diff
@@ -0,0 +1,13 @@
+export { default as BaalABI } from '../abi/contracts/Baal.sol/Baal.json';
+export { default as BaalSummonerABI } from '../abi/contracts/Baal.sol/BaalSummoner.json';
+export { default as LootABI } from '../abi/contracts/LootERC20.sol/Loot.json';
+export { default as SharesABI } from '../abi/contracts/SharesERC20.sol/Shares.json';
+export { default as PosterABI } from '../abi/contracts/tools/Poster.sol/Poster.json';
+export { default as TributeMinionABI } from '../abi/contracts/tools/TributeMinion.sol/TributeMinion.json';
+
+export { BaalFactory } from './types/BaalFactory';
+export { BaalSummonerFactory } from './types/BaalSummonerFactory';
+export { LootFactory } from './types/LootFactory';
+export { SharesFactory } from './types/SharesFactory';
+export { PosterFactory } from './types/PosterFactory';
+export { TributeMinionFactory } from './types/TributeMinionFactory';
```

Comment:
this is the main dist file for exporting abis and types into one place so they can be imported into the monorepo from `@baal-contracts`